### ISSUE TITLE
BREAKING CHANGE in v2.4

### ISSUE THREAD

**Issue by slegay** [recombee/net-api-client#4]
It looks like you are now forcing camelCasing on the JSONserializer:

RecombeeClient.cs, line 254
```
var serializerSettings = new JsonSerializerSettings();
 serializerSettings.ContractResolver = new CamelCasePropertyNamesContractResolver();
```

We uses PascalCase for our column names. Now all Recombee API calls are failing. We are reverting to 2.3 for now but this is a major issue - I would expect the column names to be used as defined without alteration, as it always was.

**Comment by OndraFiedler** [comment]
We confirm the bug. It is an unintended side-effect (the goal was to transform names of Logic parameters), which unfortunately made it through the tests.
Will fix it ASAP, please use 2.3.0 until then.

**Comment by OndraFiedler** [comment]
The issue should be fixed now. Thanks for reporting!
